File: Coding/Nicomachus Theorem/Squared_triangular_number1.cpp
// Simple C++ program to find sum of series
// with cubes of first n natural numbers

// A000537 		Sum of first n cubes; or n-th triangular number squared. 
#include <iostream>
using namespace std;

/* Returns the sum of series */
int sumOfSeries(int n)
{
	int sum = 0;
	for (int x = 1; x <= n; x++)
		sum += x * x * x;
	return sum;
}

// Driver Function
int main()
{
	//int n = 5;
	int n =0;
	cin >> n;
	cout << sumOfSeries(n) << endl;
	return 0;
}

File: Coding/Nicomachus Theorem/Squared_triangular_number1.py
# Simple Python program to find sum of series
# with cubes of first n natural numbers

#A000537 		Sum of first n cubes; or n-th triangular number squared. 

# Returns the sum of series
def sumOfSeries(n):
	sum = 0
	for i in range(1, n + 1):
		sum += i * i*i
		
	return sum


# Driver Function
#n = 5

n = int(input())

print(sumOfSeries(n))

# Code Contributed by Mohit Gupta_OMG <(0_o)>
